appveyor.yml

```diff
@@ -6,16 +6,13 @@ install:
   # Get the latest stable version of Node.js or io.js
   - ps: Install-Product node $env:nodejs_version
   - ps: $env:BuildNumber= $env:APPVEYOR_BUILD_NUMBER
+  - ps: $env:Configuration="Release"
 init:
   - git config --global core.autocrlf true
 build_script:
-  - dotnet restore
```

Comment:
what's the reason not to just for dotnet build?
I asked the MVC team and they said no to use the same scripts as them as they are doing custom things in these.